remoteworkitem/scheduler.go

```diff
@@ -80,17 +102,32 @@ func fetchTrackerQueries(db *gorm.DB) []trackerSchedule {
 func lookupProvider(ts trackerSchedule) TrackerProvider {
 	switch ts.TrackerType {
 	case ProviderGithub:
-		return &GithubTracker{URL: ts.URL, Query: ts.Query}
+		var q string
+		if ts.LastUpdated != nil {
```

Comment:
Actually, the queries are different for GitHub and Jira. Still, the else can be moved as the default value. I will make that change.